workflow/tools/util_in_docker/Dockerfile

```diff
@@ -8,11 +8,16 @@ RUN apt-get update && \
 RUN cd /usr/local/bin
 RUN ln -s /usr/bin/python3 python
 RUN pip3 install --upgrade pip
+RUN apt-get install -y software-properties-common wget
+
 
 VOLUME ["/app"]
 COPY . /app
 WORKDIR /app
 
+ADD cgc-uploader.tgz /app/
```

Comment:
why not download this file `cgc-uploader.tgz` in docker file? We shouldn't have to upload it upfront and check it into this git repo